src/test/java/org/exmple/utility/UtilityMethodsTest.java

```diff
@@ -96,4 +97,19 @@ public void countEven_shouldThrowNullPointerException() {
 
         assertTrue(actualMessage.contains(expectedMessage));
     }
+    @Test
+    public void countPositive_shouldCountPositiveNum() {
+        int[] array = {1,-1,2,-2,3,-3,4,-4,-5,5,-6,6,-7,7,-8,8,-9,9};
+
+        int result = UtilityMethods.countPositive(array);
+
+        assertEquals(9, result);
+    }
+    @Test
```

Comment:
Please, add an additional line before the method to visually separate the methods